graphqlapi/src/main/resources/schemafirst/cql_directives.graphql

```diff
@@ -114,6 +114,16 @@ directive @cql_index (
 
 ) on FIELD_DEFINITION
 
+"""
+The consistency level of the CQL SELECT generated for a query.
+"""
+enum QueryConsistency {
+    LOCAL_ONE
+    LOCAL_QUORUM
+    ALL
+    SERIAL
+    LOCAL_SERIAL
+}
```

Comment:
The available constants for each enum, and the default values, mimic the existing behavior of the CQL-first API.